manifests/init.pp

```diff
@@ -34,13 +34,13 @@
 #                               type:Optional[Stdlib::Absolutepath]
 #
 # $forward::                    The forward option
-#                               type:Optional[String]
+#                               type:Optional[Enum['only', 'first']]
```

Comment:
I'm mostly asking because we have some others below where they aren't quoted. I'd prefer using quoted only but until I saw both versions I hadn't thought about it that much.